pkg/releasesync/releasesync.go

```diff
@@ -220,6 +238,91 @@ func releaseDetailsToDB(architecture string, tag ReleaseTag, details ReleaseDeta
 	return &release
 }
 
+func parseChangeLogJSON(releaseTag string, changeLogJSON ChangeLog) models.ReleaseTag {
+	releaseChangeLogJSON := models.ReleaseTag{}
+
+	releaseChangeLogJSON.PreviousReleaseTag = changeLogJSON.From.Name
+
+	for _, c := range changeLogJSON.Components {
+		if c.Name == "Kubernetes" {
+			releaseChangeLogJSON.KubernetesVersion = c.Version
+		} else if c.Name == "Red Hat Enterprise Linux CoreOS" {
+			releaseChangeLogJSON.CurrentOSVersion = c.Version
+
+			// doesn't seem like we have all the right data to build this out
+			// https://releases-rhcos-art.apps.ocp-virt.prod.psi.redhat.com/?release=413.86.202301170928-0&stream=releases%2Frhcos-4.13
+			// releaseChangeLogJson.CurrentOSURL = fmt.Sprintf("https://releases-rhcos-art.apps.ocp-virt.prod.psi.redhat.com/?release=%s&stream=", releaseChangeLogJson.CurrentOSVersion, "???")
```

Comment:
Is there a jira somewhere with CRT tracking adding the missing fields? These URL's are important for me when I debug suspected OS issues, it shows a diff between the last OS version and the current one

Reply:
The jira tracking the URL changes upstream from release-controller is https://issues.redhat.com/browse/OCPCRT-171.  Brads changes restoring the previously missing URLs from the JSON is https://github.com/openshift/release-controller/pull/491/files.  When comparing the output it looked as though we had our urls reversed for currentOS and previous os I updated the changelog_parser to reflect that.